docs/src/environments/search-engine-visibility.md

```diff
@@ -8,7 +8,7 @@ toc: false
 When you have development environments,
 you don't want search engines indexing them and diluting the SEO of your production site.
 
-All development environments are hidden.
+Search engine indexers are told to ignore all development environments.
```

Comment:
:bulb:
Would it make sense to add a note that not all bots comply to that rule and if privacy is needed it's best to use http auth?